### ISSUE TITLE
Update "prob_type" CV

### ISSUE THREAD

**Issue by agstephens** [ukcp-data/UKCP18_CVs#18]
The "UKCP18_prob_type.json" file needs a better structure. Please change the list structure into a dictionary with contents such as:

```
{
    "percentile":{
            "label":"Percentile",
            "description":"The value below which a specified percentage falls."
}
```
Add a similar section for the "sample" prob type.

**Comment by charliepascoe** [comment]
Added "sample" section to prob_type vocab.  
Assumed sample data is for: 

- one location
- one emission scenario
- one time period

Added lists of administrative regions and river basins used in UKP09.
Added time periods and emission scenarios from the UKCP18 dataset spreadsheet https://docs.google.com/spreadsheets/d/1oNmJAYLGt6UrenY8XH_4gTyZGlFVW9plgyK75L3BN0Y/edit#gid=969822144 

Info will need to be reviewed by UKCP18 team at the Met Office.

**Comment by charliepascoe** [comment]
Updated with simplified info.
Note that I have used the UKCP09 definition of sampled data in the description field.
http://ukclimateprojections.metoffice.gov.uk/22563

The "sample" description will need to be reviewed by the UKCP18 team at the Met Office.

"The sampled data provides 10,000 samples, each of which represents a projected change in climate for one location, emission scenario and time period"
